src/exec/IO_Flow_Parameter_Set.h

```diff
@@ -22,6 +22,11 @@ class IO_Flow_Parameter_Set : public Parameter_Set_Base
 	flash_die_ID_type* Die_IDs;//Resource partitioning: which die ids are allocted to this flow
 	flash_plane_ID_type* Plane_IDs;//Resource partitioning: which plane ids are allocated to this flow
 	unsigned int Initial_Occupancy_Percentage;//Percentage of the logical space that is written when preconditioning is performed
+	// BugFix enable trace playback
+	Utils::Address_Distribution_Type Address_Distribution;
+	char Percentage_of_Hot_Region;//This parameters used if the address distribution type is hot/cold (i.e., (100-H)% of the whole I/O requests are going to a H% hot region of the storage space)
```

Comment:
These parameters are just used for synthetic message generation. They should not be part of the base message generation parameters.